src/app/content/components/Assigned.tsx

```diff
@@ -36,6 +37,10 @@ const useLoadSection = (currentSection: ArchiveTreeSection | undefined) => {
     if (!book || !currentSection) {
       return;
     }
+
+    const window = assertWindow();
+    window.scrollTo(0, 0);
```

Comment:
can we try clearing the `content.page` state when `requestPage` (with a different page than is in state already) is dispatched and see if that helps?